jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -73,6 +80,82 @@ local excludedRules = [
       { alert: 'PrometheusErrorSendingAlertsToAnyAlertmanager' },
     ],
   },
+  {
+    name: 'kube-prometheus-node-recording.rules',
+    rules: [
+      { alert: 'instance:node_cpu:rate:sum' },
+      { alert: 'instance:node_network_receive_bytes:rate:sum' },
+      { alert: 'instance:node_cpu:ratio' },
+      { alert: 'instance:node_network_transmit_bytes:rate:sum' },
+    ],
+  },
+  {
+    name: 'node.rules',
+    rules: [
+      { record: 'node:node_num_cpu:sum' },
+      { record: 'node_namespace_pod:kube_pod_info:' },
+    ],
+  },
+  {
+    name: 'openshift-build.rules',
+    rules: [
+      { record: 'build_error_rate' },
+    ],
+  },
+  {
+    name: 'openshift-kubernetes.rules',
+    rules: [
+      { record: 'cluster:capacity_cpu_cores_hyperthread_enabled:sum' },
+      { record: 'cluster:capacity_cpu_sockets_hyperthread_enabled:sum' },
+      { record: 'cluster:container_cpu_usage:ratio' },
+      { record: 'cluster:container_spec_cpu_shares:ratio' },
+      { record: 'cluster:hyperthread_enabled_nodes' },
+      { record: 'cluster:infra_nodes' },
+      { record: 'cluster:master_infra_nodes' },
+      { record: 'cluster:memory_usage:ratio' },
+      { record: 'cluster:node_cpu:ratio' },
+      { record: 'cluster:node_cpu:sum_rate5m' },
+      { record: 'cluster:usage:containers:sum' },
+      { record: 'cluster:usage:ingress_frontend_bytes_in:rate5m:sum' },
+      { record: 'cluster:usage:ingress_frontend_bytes_out:rate5m:sum' },
+      { record: 'cluster:usage:ingress_frontend_connections:sum' },
+      { record: 'cluster:usage:kube_node_ready:avg5m' },
+      { record: 'cluster:usage:kube_schedulable_node_ready_reachable:avg5m' },
+      { record: 'cluster:usage:openshift:ingress_request_error:fraction5m' },
+      { record: 'cluster:usage:openshift:ingress_request_total:irate5m' },
+      { record: 'cluster:usage:openshift:kube_running_pod_ready:avg' },
+      { record: 'cluster:usage:pods:terminal:workload:sum' },
+      { record: 'cluster:usage:resources:sum' },
+      { record: 'cluster:usage:workload:capacity_physical_cpu_core_seconds' },
+      { record: 'cluster:usage:workload:capacity_physical_cpu_cores:max:5m' },
+      { record: 'cluster:usage:workload:capacity_physical_cpu_cores:min:5m' },
+      { record: 'cluster:usage:workload:ingress_request_error:fraction5m' },
+      { record: 'cluster:usage:workload:ingress_request_total:irate5m' },
+      { record: 'cluster:usage:workload:kube_running_pod_ready:avg' },
```

Comment:
at least all of these rules are sent to Telemetry so they shouldn't be removed...

Reply:
These rules are not present in [the config file "metrics.yaml" for telemetry client](https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster-monitoring-operator_04-config.yaml).
Is there other config files we should pay attention to?